src/main/java/lotto/domain/Lottos.java

```diff
@@ -4,12 +4,40 @@
 import java.util.Map;
 
 public class Lottos {
+    public static final String NUMBER_TYPE_ERROR = "[ERROR] 숫자만 입력할 수 있습니다";
+    public static final String NUMBER_RANGE_ERROR = "[ERROR] 입력하신 금액으로 구입할 수 없습니다";
+    private static int manualCount;
+    private static int randomCount;
     private final ArrayList<Lotto> lottoGroup = new ArrayList<>();
```

Comment:
필드를 List<Lotto> 으로 선언 하는 것과 ArrayList<Lotto> 로 선언하는 것과 어떤 차이가 있을까요? 
```suggestion
    private final List<Lotto> lottoGroup = new ArrayList<>();
```

Reply:
앗 강의에서 이 부분 배우고 수정하려고 했었는데 깜빡했네요 😂